fix(weaver): add get-func-name dependency to fabric interop SDK

hollermay opened the pull request:
Adds get-func-name@2.0.0 as a direct dependency to the Fabric interoperation node SDK to resolve ESM compatibility issues that were causing test failures. This ensures proper module resolution when running tests and maintains compatibility with the SDK's existing dependencies.

**Pull Request Requirements**
- [x] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [x] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [x] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [x] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [x] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

hollermay commented:
@takeutak @petermetz kindly review my PR

sandeepnRES commented on weaver/sdks/fabric/interoperation-node-sdk/package.json:
```diff
@@ -64,6 +64,7 @@
     "fabric-common": "2.2.20",
     "fabric-network": "2.2.20",
     "fabric-protos": "2.2.20",
+    "get-func-name": "2.0.0",
```
Thanks for the PR. Let's test it in workflows.
Just one comment, does this needs to be in dependencies or devDependencies? My reason being it is not required at runtime.

hollermay replied:
It was an issue with only the dependency and with this version it checked the tests in my runtime.

sandeepnRES replied:
Does it not work, if this `get-func-name` is put in `devDependencies` of package.json?

sandeepnRES commented:
> > Also @hollermay can you make the **PR** and the **commit** title same as issue title. That's a guideline in Cacti. This makes it easy to track PRs and its corresponding issues.
> > The description to the solution you already have it in the commit message body.
> > Thank you.
> 
> I have made the changes to the title. Kindly check.

Thanks, please make the commit title same too.

sandeepnRES commented:
@hollermay You will need to sign the commit, that's why DCO check is failing.
Either use `git commit -s` when you commit, in case you have already committed and want to sign just the last commit use:
`git commit --amend --signoff`

sandeepnRES reviewed (approved):
Looks good to me, all tests are passing hence approving.
Please address the comments above before merging.

hollermay commented:
> Looks good to me, all tests are passing hence approving. Please address the comments above before merging.

I've addressed all the comments and did the changes as per your requirements. With the valid signoffs the PR is ready to be merged.

hollermay commented:
@sandeepnRES kindly review

VRamakrishna commented:
@hollermay I'll reach out to @sandeepnRES for his approval. Thanks for your diligence on this!
